pyproject.toml

```diff
@@ -115,7 +115,14 @@ exclude = ["tests/static/**"]
 
 [tool.setuptools.package-data]
 ethereum_test_forks = ["forks/contracts/*.bin"]
+"pytest_plugins.execute" = ["eth_config/networks.yml"]
+"pytest_plugins.consume" = [
+    "simulators/hive_tests/__init__.py",
+    "simulators/hive_tests/test_via_*.py",
+    "direct/test_via_direct.py"
+]
```

Comment:
I think this isn't necessary, as they are technically already in the package, no?

We don't have to include `src/pytest_plugins/execute/eth_config/execute_eth_config.py`, for example.

Reply:
I just tried it out and it works, I'm not sure what led me to think this was necessary, it was in between the many trial-and-error attempts :P

Btw, the command I'm using to test this out is:
```bash
uv run --refresh --with /home/marioevz/Development/Eth/execution-spec-tests consume rlp --input=https://github.com/ethereum/execution-spec-tests/releases/download/v4.5.0/fixtures_stable.tar.gz
```
From a folder that does not contain eest code.

Reply:
`--refresh` from a local folder! Nice! Yeah, it's important to cd out of eest root :)